index.html

```diff
@@ -180,8 +180,28 @@ <h4 class="setup-similar-title">Похожие персонажи</h4>
     <p class="setup-similar-label">Петр Петров</p>
   </div>
 </template>
-
+<script>
+  var FIREBALL_SIZE = 22;
+  var WIZARD_SPEED = 3;
+  var WIZARD_WIDTH = 70;
+  var WIZARD_PROP = 1.337;
+
+  var getFireballSpeed = function (left) {
+    return left ? 5 : 2;
+  };
+
+  var getWizardHeight = function (){
+    return WIZARD_PROP * WIZARD_WIDTH;
+  };
+
+  var getWizardX = function (width) {
+    return width - ((width/2) + WIZARD_WIDTH/2);
+
+  };
+  var getWizardY = function (height) {
+    return height - ((height/3) + getWizardHeight ());
```

Comment:
Перемудрил, твоя запись эквивалентна
```suggestion
    return height*2/3 - getWizardHeight ();
```